src/Umbraco.Tests/Persistence/Repositories/DocumentRepositoryTest.cs

```diff
@@ -557,12 +558,12 @@ public void UpdateContentWithNullTemplate()
                 var repository = CreateRepository((IScopeAccessor)provider, out _);
 
                 var content = repository.Get(NodeDto.NodeIdSeed + 2);
-                content.Template = null;
+                content.TemplateId = 0;
```

Comment:
again, null should be used, and the we also need to update the assert to use `Assert.IsNull`